### ISSUE TITLE
in mobile view each blog using brick_skin shows the same header picture

### ISSUE THREAD

**Issue by farfallosa** [b2evolution/bricks_skin#4]
may four blogs with different headers etc. set up in skin bricks_skin work pretty well on desktop. But choosing the skin on my mobile phone instead of touch_skin all four blogs show the same header picture and a configuration (showing breadcrump, overlay, title) I don't use in any of the blogs by customizing.
So where can I customize the header performance for mobile devices?

To get support would really be helpful
Regards, will

**Comment by Ariflaw** [comment]
Hi @farfallosa can you please send the screenshots?

**Comment by farfallosa** [comment]
Hi @Ariflaw 
here comes the screenshot. (The first line  shows screenshots made by a mobile phone. The second line shows a redsign check on desktop using firefox tools; i case you want to test it by yourself here are the Urls: tourismus.genba.org, wegscheid.genba.org, linz.genba.org, fokus.genba.org)
[Meanwhile I deleted all the preset headers. Now the header image is gray although it is customized and should show own header images.]

Also the EU Cookie consent does not show up in mobile view. So there seems to be a legal probem, too (almost in EU)

![responsive_bricks_skin_b2e_bug](https://user-images.githubusercontent.com/11560452/28261952-93f71d9a-6ae1-11e7-9bd9-ba2ace9b2ef9.jpg)

Would be great to get a fix

**Comment by Ariflaw** [comment]
@farfallosa I've updated for the header title issue, please update the code and see that is fixed or still have an issue.

Thanks

**Comment by farfallosa** [comment]
@Ariflaw I updated all those files you updated recently. I cleared all caches also the browser ones and started with another mobile browser (chrome instead of firefox) nothing changed. The issue still exists.
(but what happed by updating style.css and style.min.css: my customized css ceased, lokated in rsc/css/bootstrap/custom_94110.css)
You can check it. You find the url addresses mentioned above.

Thanks

**Comment by Ariflaw** [comment]
Hi @farfallosa I see the new code I've add not load properly. Maybe you need to reinstall the skin to have an update.

**Comment by farfallosa** [comment]
Hi @Ariflaw 
Tried to rename the updated bricks_skin and to install this for testing. (bricks_skin2)
I set folder rights to 0755; I edited the _skin.class  class to: `class bricks_Skin2 extends Skin` but nevertheless installing this skin clone failed I always get this: 
![b2evo_2017-07-20_11-05-24](https://user-images.githubusercontent.com/11560452/28409644-7cb4ff42-6d3b-11e7-9484-a0df79883035.jpg)

I checked the b2e forum and manual. But I can not figure out, how to fix it and get the updated clone of bricks_skin running.
Would you please be so kind and help me?  (In a next step I'd check the mobile view)

Regards

**Comment by miloszivojinovic** [comment]
Hi @farfallosa . Regarding this question:

> But I can not figure out, how to fix it and get the updated clone of bricks_skin running.

If you wanted to create a separate skin out of "Bricks Skin" for testing purposes, this is how you do it properly:

- first, you rename skin folder to `b2evo_installation/skins/brcks2_skin` (note that all skin folders *must* end with `_skin` in order to work properly)
- second, you change the skin name in `_skin.class.php`, here: https://github.com/b2evolution/bricks_skin/blob/master/_skin.class.php#L39.

Then your skin Bricks2 will work.

**Comment by farfallosa** [comment]
@miloszivojinovic thanks I got my mistake. (I put the 2 after skin and not after bricks)

@Ariflaw Now the actual skin is activated for http://tourismus.genba.org. But the bug persists. The customized header image does not show up. Check it. I tried with Android.
The new version shows an additional problem. When resizing the page (header transparent) the white font changes in grey, by staying transparent. Therefore the title is unreadable.

![conrad_02 video tourismus genba_2017-08-06_13-44-20](https://user-images.githubusercontent.com/11560452/29002922-9e2d4498-7aad-11e7-95b0-ca61a2eddc1d.jpg)
![conrad_02 video tourismus genba_2017-08-06_13-44-53](https://user-images.githubusercontent.com/11560452/29002924-a646fa02-7aad-11e7-955e-47a26deb14f4.jpg)
